includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2193,6 +2201,9 @@ public function update_order_status() {
 			}
 
 			$payment_method_id = isset( $_POST['payment_method_id'] ) ? wc_clean( wp_unslash( $_POST['payment_method_id'] ) ) : '';
+			if ( 'null' === $payment_method_id ) {
+				$payment_method_id = '';
+			}
```

Comment:
See https://github.com/Automattic/woocommerce-payments/pull/3642/files#r786451779

Reply:
Should we try to confirm or fix this bug before merging the PR?

Reply:
I'll see if it's still happening before merging and report back 🤔

Reply:
Just confirmed that this is still an issue. Very easy to reproduce;

Just run a payment. If you set a breakpoint here and inspect the value you'll see that it comes back as `"null"` 🤷‍♂️